os/Kconfig

```diff
@@ -268,7 +268,14 @@ config PASS1_OBJECT
 	default ""
 
 endif # Build 2-pass
-endmenu # Build Setup
+
+config BINARY_VERSION
+	int "Binary Version(YYMMDD)"
```

Comment:
Actually not all culture uses YYMMDD, for example england culture uses DDMMYY.
(so in this case binary manager occure error??)
So you should change description about this.
For example, Binary version(default value follows YYMMDD type, please make sure that high number of version is latest version) blah blah blah blah
As per my experience, this is very serious agenda in product level.